apps/wistia-videos/src/components/Field.tsx

```diff
@@ -107,38 +137,51 @@ const Field = (props: FieldProps) => {
         </Paragraph>
       ) : (
         <>
-          {data?.length > 0 && !error ? (
+          {data?.length && !error ? (
```

Comment:
You dont need the `?` now. You can just make it `data.length`

Reply:
Fixed 👍